frontend/components/vault/Vault.vue

```diff
@@ -0,0 +1,278 @@
+            <button
+                class="
+                    w-full
+                    border-2 border-gray-300
+                    p-1
+                    border-t-0
+                    text-center text-gray-400
+                    dark:border-gray-600 dark:text-gray-600
+                "
+                @click="toggleExpandable"
+            >
+                {{ isTableExpanded ? 'Hide additional info' : 'Show additional info' }}
+            </button>
+            <template v-if="isExplanationsShown">
+                <TextBlock class="mt-4">
+                    <template v-if="!error">
+                        The vault
+                        <format-address type="address" :value="vaultTransaction.address" shorten disable /> contains
+                        <format-currency
+                            :value="vaultTransaction.collateralAmount"
+                            :currency="vaultTransaction.collateralType"
+                        />. The initial debt taken by the vault is
+                        <format-currency :value="vaultTransaction.initialDebtDai" currency="DAI" />. The current
+                        proximity to liquidation is
+                        <AnimatedNumber :value="vaultTransaction.proximityToLiquidation" />. The next price update will
+                        happen in <TimeTill :date="vaultTransaction.nextPriceChange" />.
+                    </template>
+                </TextBlock>
+            </template>
+            <TextBlock>
+                <div class="flex w-full justify-end flex-wrap mt-4">
+                    <Tooltip :title="error" placement="bottom">
+                        <div>
+                            <Button :disabled="!!error" type="primary" class="w-60 ml-4" @click="$emit('liquidate')">
+                                Liquidate
+                            </Button>
+                        </div>
+                    </Tooltip>
+                </div>
+            </TextBlock>
+        </div>
+        <div v-else-if="vaultTransaction && vaultTransaction.state === 'liquidated'">
+            <Alert show-icon type="warning">
+                <div slot="message">
+                    <p>This vault has been liquidated into a collateral auction</p>
+                    <div class="flex justify-end mt-2">
+                        <nuxt-link :to="vaultLink">
+                            <Button type="primary"> View collateral auction {{ vaultTransaction.auctionId }} </Button>
+                        </nuxt-link>
+                    </div>
+                </div>
+            </Alert>
+            <TextBlock class="mt-4">
+                This vault was liquidated <TimeTill :date="vaultTransaction.liqudiationDate" /> in the transaction
+                <FormatAddress :value="vaultTransaction.transactionHash" />.
+            </TextBlock>
+        </div>
+        <Loading v-else-if="areVaultsFetching" is-loading class="w-full self-center Loading h-48" />
```

Comment:
What happens on `v-else`? Does page stays empty?

Reply:
Nothing will be displayed. This is the same logic we used in all the Auction Components too, so I took it over:

https://github.com/sidestream-tech/unified-auctions-ui/blob/main/frontend/components/auction/collateral/CollateralAuction.vue#L262